README.md

```diff
@@ -1,2 +1,35 @@
 # dic
 dic - dic isn't conky
+
+# Concepts
+
+## Subsistems:
+
+### 1. Receiving telemetry through:
+
+* Unix-socket
+* Pipe
+* Return code of the third-party program
+
+### 2. Preparation of telemetry:
+
+* filtering
+* normalization
+* sorting
+* approx (?)
+* etc.
+
+### 3. Output to the screen:
+
+* plot
+* diagram
+* plain text
```

Comment:
> multiple ways of outputting stuff
Yes I think it's would be useful for embedding our dashboard into another software. This a will also add to description.